protocol/0062-SPAM-spam_protection.md

```diff
@@ -83,6 +83,17 @@ Further, each party is allowed to submit up to `n` transactions per epoch where
 
 **Note** `spam.protection.max.updatePartyProfile` must be an integer greater than or equal to `0` (and default to `5`).
 
+### Transaction Spam
+
+Network parameter: `spam.order.MinimalMarginQuantumMultiple` (between 0 and infinite)
+
+If the maintenance margin for a given transaction is smaller than the parameter `spam.order.MinimalMarginSizeQuantumMultiple`, then the transaction is pre-block rejected.
+
+If the market doesn't exist or hasn't been enacted yet (and thus the maintenance margin cannot be computed), the transaction is rejected.
```

Comment:
Bit of a terminology nitpick but I'm not sure about "hasn't been enacted yet."

I think a "pending" market which is in an opening auction un till the enactment timestamp is reached still needs to be able to accept orders?

Reply:
True, I was referring to undefined markets where the margin can't be calculated yet. Will clarify that